### PR TITLE
Renamed SWC-Blueprint to NeuroBlueprint

### LINKED ISSUE #34: Consider renaming to NeuroBlueprint
The "SWC" in the name may give the impression that the specification is only meant for one institute. "Neuro" makes it more general.

### PR DESCRIPTION
**Pull request by niksirbi**
Closes #34 
I renamed all mentions of `SWC-Blueprint` to `NeuroBlueprint`.
The HTML base URL now directs to `neuroblueprint.neuroinformatics.dev` instead of `swc-blueprint.neuroinformatics.dev`. @adamltyson could you enable this domain?

### CODE REVIEW

**Comment by niksirbi** [comment]
This GitHub repository has to be renamed as well, but I don't have the required permissions. @JoeZiminski?

**Comment by JoeZiminski** [comment]
Hey @niksirbi built locally and looks good to me! Thanks I can change the repo name, maybe I will wait until this is merged before renaming the repo in case it causes some problems with the PR, actually no idea what would happen!

**Review by adamltyson — APPROVED** [review]

**Comment by adamltyson** [comment]
All done, thanks @niksirbi!

**Pull request merged**
